src/vstest.console/Processors/EnableBlameArgumentProcessor.cs

```diff
@@ -133,7 +136,8 @@ public void Initialize(string argument)
         {
             bool isDumpEnabled = false;
 
-            if (!string.IsNullOrWhiteSpace(argument) && argument.Equals(Constants.BlameCollectDumpKey, StringComparison.OrdinalIgnoreCase))
+            var parseSucceeded = LoggerUtilities.TryParseLoggerArgument(argument, out string loggerIdentifier, out Dictionary<string, string> parameters);
```

Comment:
We need to have validation here in case any invalid args, parameters passed. Check comment in UTs for detail.

Reply:
done